docs/pages/management/operations/ca-rotation.mdx

```diff
@@ -1,261 +1,466 @@
-
-Execute the transition from `init` to `update_clients`:
-
-```code
-$ tctl auth rotate --manual --type=<Var name="type" description="Certificate authority to rotate"/> --phase=update_clients
-# Updated rotation phase to "update_clients". To check status use 'tctl status'
-$ tctl status
-# Cluster  acme.cluster
-# Version  (=teleport.version=)
-# Host CA  rotating clients (mode: manual, started: Sep 20 2023 01:44:36 UTC, ending: Sep 21 2023 07:44:36 UTC)
-```
-
-<Admonition type="note">
-  Clients will temporarily lose connectivity during Proxy and Auth Server
-  restarts.
-</Admonition>
-
-Verify that nodes have caught up and now see the current cluster state:
-
-```code
-$ tctl get nodes --format=json | jq '.[] | {hostname: .spec.hostname, rotation: .spec.rotation.state, phase: .spec.rotation.phase}'
-{
-  "hostname": "terminal",
-  "rotation": "in_progress",
-  "phase": "update_clients"
-}
-```
-
-**Update servers**
-
-Now that all nodes have caught up, execute the transition from `update_clients`
-to `update_servers`:
-
-```code
-$ tctl auth rotate --manual --type=<Var name="type" description="Certificate authority to rotate"/> --phase=update_servers
-# Updated rotation phase to "update_servers". To check status use 'tctl status'
+In this example, the Teleport instance named `terminal` has updated its status
+to phase `init`. This means it has downloaded a new CA public key and is ready
+for state transitions.
 
-$ tctl status
-# Cluster  acme.cluster
-# Version  (=teleport.version=)
-# Host CA  rotating servers (mode: manual, started: Sep 20 2023 01:44:36 UTC, ending: Sep 21 2023 07:44:36 UTC)
-```
+You can use the `tctl get` command with the following resources to determine
+the rotation state of the `host` CA on each agent kind:
 
-<Admonition type="note">
-  Usually if things go wrong, they go wrong at this transition. If you have lost
-  connectivity to nodes, [roll back](#rollback) to the old certificate
-  authority.
-</Admonition>
+| Role                | `tctl get` value |
+|---------------------|------------------|
+| Application Service | `app_server`     |
+| Database Service    | `db_server`      |
+| Kubernetes Service  | `kube_server`    |
+| Proxy Service       | `proxies`        |
+| SSH Service         | `nodes`          |
 
-Verify that nodes have caught up:
+During each phase of the `host` CA rotation, make sure all Agents and Proxy
+Service instances have completed the transition to target phase before
+proceeding to the next phase. We will explain the phases in [Step
+2](#step-2-4-start-a-manual-rotation).
 
-```code
-$ tctl get nodes --format=json | jq '.[] | {hostname: .spec.hostname, rotation: .spec.rotation.state, phase: .spec.rotation.phase}'
-{
-  "hostname": "terminal",
-  "rotation": "in_progress",
-  "phase": "update_servers"
-}
-```
+If you are joining Teleport processes to a cluster via the Teleport Auth Service
+using a [join token](../../agents/join-services-to-your-cluster/join-token.mdx),
```

Comment:
```suggestion
If you are joining Teleport processes to a cluster by dialing the Teleport Auth Service directly,
```

Not sure what the join token has to do with this. All joining requires a join token.